libs/ledger-live-common/src/exchange/swap/logic.ts

```diff
@@ -23,232 +9,3 @@ export const getSwapSelectableCurrencies = (rawProviderData: Array<AvailableProv
   });
   return uniq<string>(ids);
 };
-
-// TODO deprecated when noWall
-export const getCurrenciesWithStatus = ({
```

Comment:
:clap: :clap: :clap: :clap:

Reply:
Yeah this whole file was unused except the top function :)